src/components/Configure/state/HydratedRevisionContext.tsx

```diff
@@ -117,9 +125,25 @@ export function HydratedRevisionProvider({
   if (isError(ErrorBoundary.HYDRATED_REVISION, errorIntegrationIdentifier)) {
     const intNameOrId = integrationObj?.name || integrationId || 'unknown integration';
     const errorMsg = `Error retrieving integration details for '${intNameOrId
-    }. This is sometimes caused by insufficient permissions with your credentials'`;
-
-    return <ComponentContainerError message={errorMsg} />;
+    }. This is sometimes caused by insufficient permissions with your credentials. ' 
+    ${readableErrorMsg ? `: ${readableErrorMsg}` : ''}`;
+
+    return (
+      <ComponentContainerError message={errorMsg}>
+        <div style={{
+          display: 'flex', flexDirection: 'column', gap: '1rem', paddingTop: '1rem',
+        }}
+        >
+          {connectionError && <InnerErrorTextBox message={connectionError} />}
+          <RemoveConnectionButton
+            buttonText="Remove Connection"
```

Comment:
this text should say "Try again".  "remove connection" is confusing, since even though we have a connection created in the backend at this point, the user doesn't know that and from their perspective the connection wasn't successful yet.